web/src/beta/features/Modals/AssetModal/index.tsx

```diff
@@ -242,15 +241,32 @@ const AssetListWrapper = styled.div`
   display: flex;
   flex-direction: column;
   overflow-y: auto;
+  transition: -webkit-scrollbar 1s;
+  ::-webkit-scrollbar {
+    display: none;
+  }
+  :hover::-webkit-scrollbar {
+    display: block;
+    width: 8px;
+    height: 33px;
+  }
+  ::-webkit-scrollbar-track {
+    background-color: transparent;
+    background-color: red;
+    display: none;
+  }
+  ::-webkit-scrollbar-thumb {
+    border-radius: 5px;
+    background-color: ${({ theme }) => theme.bg[2]};
+  }
```

Comment:
I think the scrollbar is taking up space so when it is not showing it looks like there is more padding on the right than the left